templates/component_model.yaml

```diff
@@ -221,6 +221,16 @@ methods:
       type: MAP(XTYPE(InterfacePtr), VECTOR(XTYPE(InterfacePtr)))
     description: "This functions resolves any interfaces of inner parts which do not match any of the parts' model interfaces and a list of possible future matches."
 
+  export_inner_interface:
+    arguments:
+      - name: inner_interface
+        type: XTYPE(Interface)
+      - name: with_empty_facts
```

Comment:
Assign a default value here

Reply:
Based on the previous comment, it should probably be `false` as default?